packages/containerd/containerd-cri-base-json

```diff
@@ -0,0 +1,151 @@
+{
+    "ociVersion": "1.0.2-dev",
```

Comment:
confirm if this is the latest OCI spec version from an upstream source, not just `ctr oci spec`

Reply:
yes it has been 1.0.2-dev since 2020
https://github.com/opencontainers/runtime-spec/blob/main/specs-go/version.go